multiversx/mx-chain-go
# update node's dockerfile

## Reasoning behind the pull request
- the docker file was missing `RUN apt-get update && apt-get upgrade -y`
  
## Proposed changes
- added `RUN apt-get update && apt-get upgrade -y`

## Testing procedure
- in the root of the repository, run `docker build . --file docker/node/Dockerfile` - it should work

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
